libs/remix-ui/run-tab/src/lib/reducers/runTab.ts

```diff
@@ -115,14 +115,14 @@ export const runTabInitialState: RunTabState = {
     providerList: [{
       id: 'vm-mode-london',
       dataId: 'settingsVMLondonMode',
-      title: 'Execution environment does not connect to any node, everything is local and in memory only.',
+      title: 'Execution environment does not connect to any node. All data is stored in browser memory only.',
```

Comment:
I think it can be like: `Environment local to Remix. Data is stored in browser and deployed contracts disappear on reload`